05/app.js

```diff
@@ -9,6 +9,57 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+//na spotkanku mam kilka pytań odośnie tego zadania :). Podejrzewam ze i tak do poprawy :<
+
+const linkElement = document.querySelectorAll('.link');
+const paragraphElement = document.querySelectorAll('.text');
+let numberP1 = 1;
+let clickParagaphCounter = 1;
+let counter = 1;
+
+const clickCounter = function(){
+
+    if (this.dataset.id === 'p1'){
```

Comment:
Chodziło o bardziej uniwersalne rozwiązanie. Jesli klikasz w pierwszy paragraf to jest jak zrobiłeś, ale potrzeba obsługi drugiego i trzeciego.